angular-hub/src/app/pages/communities/index.page.ts

```diff
@@ -50,6 +49,14 @@ export const routeMeta: RouteMeta = {
         <li>
           <app-community-card [community]="community"></app-community-card>
         </li>
+      } @empty {
```

Comment:
please remove this part as the country list is based on the existing communities, so we can't get 'no result' for this page.